client/src/components/SudoModePasswordField/SudoModePasswordField.scss

```diff
@@ -0,0 +1,55 @@
+// entwine component before the react component has loaded
+// styles are set to prevent a FOUT
+.SudoModePasswordField {
+  min-height: 108px;
+
+  @include media-breakpoint-up(lg) {
+    min-height: 140px;
+  }
+
+  .form__field-holder input {
+    display: none;
+  }
+}
```

Comment:
We tend to use a loading component for this - why are we doing it differently here?

Reply:
I'm not sure what you are referring to here

Reply:
NEvermind, not worth arguing over